data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report4_dateshift.py

```diff
@@ -597,23 +597,21 @@
 
 
 SELECT * FROM df1
-JOIN df2 USING(n_row_not_pass)
-JOIN df3 USING(n_row_not_pass)
-JOIN df4 USING(n_row_not_pass)
-JOIN df5 USING(n_row_not_pass)
-JOIN df6 USING(n_row_not_pass)
-JOIN df7 USING(n_row_not_pass)
-JOIN df8 USING(n_row_not_pass)
-
-
+UNION ALL SELECT * FROM df2 
+UNION ALL SELECT * FROM df3 
+UNION ALL SELECT * FROM df4  
+UNION ALL SELECT * FROM df5  
+UNION ALL SELECT * FROM df6  
+UNION ALL SELECT * FROM df7  
+UNION ALL SELECT * FROM df8 
 
 """)
 q = query.render(project_id=project_id,
                  pipeline=pipeline,
                  com_cdr=com_cdr,
                  deid_cdr=deid_cdr)
 df1 = execute(client, q)
-if df1.eq(0).any().any():
+if df1['n_row_not_pass'].sum() == 0:
```

Comment:
Thanks for the suggestion here! Good catch!

As for the query updates. I tend toward not wanting to update them.
To keep the query numbers consistent with previous notebook results. Also it could be a kind of paper trail for removed queries. Because there is no query2 we can assume it was removed. 
Do you agree or think it better have a more clean notebook?

Reply:
It would be helpful to keep them as they are, so we can compare with previous notebooks.